### ISSUE TITLE
[BUG] can not create a docker based rancher install with k3s 1.25.9

### ISSUE THREAD

**Issue by corbosman** [rancher/rancher#41349]
**Rancher Server Setup**
- Rancher version: 2.7.3
- Installation option (Docker install/Helm Chart): Docker

**User Information**
- What is the role of the user logged in? Admin

**Describe the bug**
When I use the docker based rancher install, then in the UI create a k3s cluster version 1.25.9, I can not successfully add a machine to the cluster by copy-pasting the necessary command in an ubuntu 22.04 server. It's stuck in an endless loop. I can install 1.25.7 just fine on the same machine. 

I ended up there because I tried upgrading a cluster from 1.25.7 to 1.25.9 which also failed.

**Comment by corbosman** [comment]
I was able to install/add a 1.25.9 on a completely new VM. Something in my old VM, which was fully up to date with 22.04 caused it to fail.  I'll close this ticket since it's probably unlikely I'm going to find out what that is.

**Comment by corbosman** [comment]
I'm reopening to make sure this isn't an issue. I was able to make a clone of my failing VM and tried a few more times to see why it failed installing a 1.25.9 k3s cluster. I checked the output of the failing and succeeding VMs line by line, and noticed the failing VM was missing iptables. Mind you, it's been running without iptables for over a year and been through several rancher and k8s upgrades, but somehow as of 1.25.9 (and not 1.25.7) it wants to run iptables during the install/upgrade. Is iptables a requirement? 


```Apr 29 12:41:13 k3s rancher-system-agent[1223]: time="2023-04-29T12:41:13Z" level=info msg="[65c12025d4ca1567651b1d8afcb293eeaebc8865a4c14d4ec9ec71d2eb17d339_0:stdout]: /usr/sbin/iptables-save"```

vs

```
Apr 29 21:08:06 k3s4 rancher-system-agent[1430]: time="2023-04-29T21:08:06+02:00" level=info msg="[49403717dc623cf9b924a06cf23cc4fa5f9607ee9123f877d9fdb9bf4eeb1809_0:stderr]: /var/lib/rancher/agent/work/20230429-210803/49403717dc623cf9b924a06cf23cc4fa5f9607ee9123f877d9fdb9bf4eeb1809_0/installer.sh: 972: iptables-save: not found"
```

**Comment by Oats87** [comment]
Yes, it appears `iptables` is a relatively new requirement stemming from the K3s side: https://github.com/k3s-io/k3s/pull/727

**Comment by corbosman** [comment]
Did you mistype the issue? This seems to be the same problem on the k3s repo:  https://github.com/k3s-io/k3s/issues/7291 . It seems recently k3s has added iptables commands to their scripts which apparently breaks rancher upgrades and installs of 1.25.9.
